Detectors/MUON/MCH/Raw/Common/include/MCHRawCommon/SampaCluster.h

```diff
@@ -43,15 +43,15 @@ struct SampaCluster {
   ///
   /// if some parameter does not fit within its expected range
   /// a std::invalid_argument exception is thrown.
-  explicit SampaCluster(uint16_t timestamp, uint32_t chargeSum);
+  explicit SampaCluster(uint16_t timestamp, uint32_t bunchCrossing, uint32_t chargeSum);
```

Comment:
maybe for consistency we should rename the first parameter (and corresponding data member) to "sampaTime" ?

Reply:
Done.